base/cvd/cuttlefish/host/commands/cvd/server_command/create.cpp

```diff
@@ -107,36 +130,46 @@ Result<CreateFlags> ParseCommandFlags(const cvd_common::Envs& envs,
       .product_path = DefaultProductPath(envs),
       .start = true,
       .config_file = "",
+      .acquire_file_locks = GetAcquireFileLockEnvValue(envs).value_or(true),
   };
   std::vector<Flag> flags = {
       GflagsCompatFlag("host_path", flag_values.host_path),
       GflagsCompatFlag("product_path", flag_values.product_path),
       GflagsCompatFlag("start", flag_values.start),
       GflagsCompatFlag("config_file", flag_values.config_file),
+      GflagsCompatFlag(kAcquireFileLock, flag_values.acquire_file_locks),
   };
   CF_EXPECT(ConsumeFlags(flags, args));
   return flag_values;
 }
 
 CommandRequest CreateLoadCommand(const CommandRequest& request,
-                                   cvd_common::Args& args,
-                                   const std::string& config_file) {
-  return CommandRequest()
-      .SetEnv(request.Env())
-      .AddArguments({"cvd", "load"})
-      .AddArguments(args)
-      .AddArguments({config_file});
+                                 cvd_common::Args& args,
+                                 const std::string& config_file) {
+  auto res = CommandRequestBuilder()
+                 .SetEnv(request.Env())
+                 .AddArguments({"cvd", "load"})
+                 .AddArguments(args)
+                 .AddArguments({config_file})
+                 .Build();
+  CHECK(res.ok()) << "Failed to build load command request: "
+                  << res.error().FormatForEnv();
+  return res.value();
 }
 
 CommandRequest CreateStartCommand(const CommandRequest& request,
-                                    const selector::LocalInstanceGroup& group,
-                                    const cvd_common::Args& args,
-                                    const cvd_common::Envs& envs) {
-  return CommandRequest()
-      .SetEnv(envs)
-      .AddArguments({"cvd", "start"})
-      .AddArguments(args)
-      .AddSelectorArguments({"--group_name", group.GroupName()});
+                                  const selector::LocalInstanceGroup& group,
```

Comment:
The only caller of `CreateLoadCommand` and `CreateStartCommand` already returns a `Result`, so can these functions error out with `Result`s as well?

Reply:
Done